service/src/main/java/uk/gov/hmcts/reform/fpl/config/utils/EmergencyProtectionOrderDirectionType.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.fpl.config.utils;
+
+public enum EmergencyProtectionOrderDirectionType {
+
+    CONTACT_WITH_NAMED_PERSON("Contact with any named person"),
+    CHILD_MEDICAL_ASSESSMENT("A medical or psychiatric examination, or another assessment of the child"),
+    MEDICAL_PRACTITIONER("To be accompanied by a registered medical practitioner, nurse or midwife"),
+    EXCLUSION_REQUIREMENT("An exclusion requirement"),
+    OTHER("Other direction relating to an emergency protection order");
+
+    private final String label;
+
+    EmergencyProtectionOrderDirectionType(String label) {
```

Comment:
Please make private.

Reply:
private is redundant for enum contructors

Reply:
You are right 👍